src/lib.rs

```diff
@@ -400,6 +403,18 @@ impl<'tree, Tab> DockArea<'tree, Tab> {
                                     Sense::click_and_drag()
                                 };
 
+                                response.0.clone().context_menu(|ui| {
```

Comment:
We should be able to disable the context menu since not every app needs it.

Add `show_context_menu: bool` to `Style` and `StyleBuilder` and use it here.